dffml/cli/dataflow.py

```diff
@@ -104,6 +107,15 @@ async def run(self):
                     for val, def_name in self.seed
                 ]
                 dataflow.seed.extend(self.seed)
+
+                # flow argument key is of the form opname.inputs/conditions.keyname
+                for v, k in self.flow:
+                    opname, val_type, key = k.split(".")
```

Comment:
We need to handle the case where there are more than three `.`, possible solution is to split, use last two, join rest of leading to use as first one.

Reply:
Let's use `split_dot_separated` here.

Reply:
Let's add support for setting config via:

```python
                for v, k in self.config:
                    k = split_dot_separated (k)
                    traverse_set(dataflow.config, k, v)
```